internal/kube/client.go

```diff
@@ -24,6 +24,8 @@ import (
 	"k8s.io/client-go/rest"
 	"k8s.io/client-go/restmapper"
 	"k8s.io/client-go/tools/clientcmd"
+
+	fluxClient "github.com/fluxcd/pkg/runtime/client"
```

Comment:
Any specific reason for this alias?

Reply:
I was aligning it with the other reference in the project, but given that there is no other `/client` here, I removed that altogether.